portfolio/src/main/webapp/script.js

```diff
@@ -61,13 +61,25 @@ function createCommentBox(comment) {
 
     const nameElement = document.createElement('h3');
     nameElement.style.textAlign = 'left';
+    nameElement.style.display = 'inline';
     nameElement.innerHTML = comment.name;
 
+    const usernameElement = document.createElement('h3');
+    usernameElement.style.textAlign = 'left';
```

Comment:
It might be easier to give the element an id or class in JS and adding styling in the CSS file.